README.md

```diff
@@ -150,6 +129,79 @@ If CallKit has been set up correctly, Tom's device will ring like a phone call,
 
 Note: Due to an issue with ARKit, you cannot start a Remote Streem from a view using the camera https://forums.developer.apple.com/message/411888#411888
 
+#### Roles
+
+Roles dictate which side of the streem your on as in whether you're sharing your video or receiving video. They also affect which tools are available to you. The different roles are:
+
+* `LOCAL_CUSTOMER`: This is the primary role for the customer in a two-way streem
+* `ONSITE_CUSTOMER`: This is the primary role for a customer in an onsite (one-way) streem 
+* `REMOTE_PRO`: This is the primary role for the pro in a two-way streem
+* `ONSITE_PRO`: This is the primary role for a pro in an onsite (one-way) streem
+
+In general if you are starting a remote streem you will want the `LOCAL_CUSTOMER` role. 
+
+### Getting Remote User IDs
+
+In order to start a remote streem, your app will need to supply the `remoteUserId` of the user your app's user is going to streem with. There are two mechanisms available in the SDK for retrieving a `remoteUserId`. The first is through getting a list of recently logged in users. The second is through an invitation. In addition to these two methods, you can maintain your own list of `remoteUserIds` and supply them to your app however you choose. 
+
+#### Recently Logged In Users
+
+By calling the SDK's `getRecentlyIdentifiedUsers` method on the Streem `sharedInstance` you will receive a list of recently logged in users for your company. You can use this method to filter only experts. You then give this method a callback where you select the remote user through some mechanism and start the streem. 
```

Comment:
I thought `getRecentlyIdentifiedUsers()` was meant to be an internal function just for us to use from StreemNow. Are we actually wanting to support this for our users? If you've already discussed this with @seanadkinson, then whatever you agreed on is fine with me 😄 I just want to make sure the conversation about this has happened, since in the past we had definitely treated this as internal-only.

Reply:
StreemNow _is_ our example app, for the moment. It illustrates the use of the SDK's core functionality, while mostly avoiding authentication code.

So I think for now it makes sense to provide this minimal documentation of `getRecentlyIdentifiedUsers()`.

@seanadkinson do you have a different vision, for the near-term?

Reply:
StreemNow _is_ our example app, for the moment. It illustrates the use of the SDK's core functionality, while mostly avoiding authentication code.

So I think for now it makes sense to provide this minimal documentation of `getRecentlyIdentifiedUsers()`.

@seanadkinson do you have a different vision, for the near-term?

Reply:
We didn't really discuss, but I don't have a burning desire say we can't support this method.  It's not like this would be particularly useful in a production deployment, so its existence it illustrate functionality in a quick way, without the need for the customer to create integrations with their database, seems okay.

Reply:
👍 I hadn't realized this might be on the chopping block. From an implementer standpoint I agree with Sean that providing this as a quick way to get up and running is useful.